src/main/java/Lession3/main.java

```diff
@@ -0,0 +1,20 @@
+package Lession3;
+
+public class main {
```

Comment:
Class name from capital ;) But secondly, you should move this kind of test to dedicated Test class